public/app/plugins/panel/canvas/components/connections/ConnectionSVG2.tsx

```diff
@@ -0,0 +1,596 @@
+                    strokeWidth={strokeWidth}
+                    markerEnd={markerEnd}
+                    markerStart={markerStart}
+                    strokeDasharray={lineStyle}
+                    strokeDashoffset={1}
+                    x1={x1}
+                    y1={y1}
+                    x2={x2}
+                    y2={y2}
+                    cursor={connectionCursorStyle}
+                  >
+                    {shouldAnimate && (
+                      <animate
+                        attributeName="stroke-dashoffset"
+                        values={getAnimationDirection()}
+                        dur="5s"
+                        calcMode="linear"
+                        repeatCount="indefinite"
+                        fill={'freeze'}
+                      />
+                    )}
+                  </line>
+                  {/* initial midpoint */}
+                  {isSelected && (
+                    <circle
+                      id={CONNECTION_VERTEX_ADD_ID}
+                      data-index={0}
+                      cx={midpoint.x}
+                      cy={midpoint.y}
+                      r={4}
+                      stroke={strokeColor}
+                      className={styles.addVertex}
+                      cursor={'crosshair'}
+                      pointerEvents="auto"
+                    />
+                  )}
+                </g>
+              )}
+            </g>
+          );
+        })
+    );
+  };
+
+  return (
+    <>
+      <svg ref={setConnectionsSVGRef} className={styles.connection}>
+        <defs>
+          <marker
+            id={EDITOR_HEAD_ID}
+            markerWidth="10"
+            markerHeight="7"
+            refX="10"
+            refY="3.5"
+            orient="auto"
+            stroke={defaultArrowColor}
+          >
+            <polygon points="0 0, 10 3.5, 0 7" fill={defaultArrowColor} />
+          </marker>
+        </defs>
+        {/* svg line for connection creation */}
+        <line
+          ref={setLineRef}
+          stroke={defaultArrowColor}
+          strokeWidth={2}
+          markerEnd={`url(#${EDITOR_HEAD_ID})`}
+          style={{ display: 'none' }}
+        />
+
+        {/* svg circle for initial vertex?
+          path? is it for the line drag handling? */}
+        <path
+          ref={setVertexPathRef}
+          stroke={defaultArrowColor}
+          strokeWidth={2}
+          strokeDasharray={'5, 5'}
+          fill={'none'}
+          style={{ display: 'none' }}
+        />
+        <circle
+          ref={setVertexRef}
+          stroke={defaultArrowColor}
+          r={4}
+          className={styles.vertex}
+          style={{ display: 'none' }}
+        />
+
+        {renderConnections()}
+      </svg>
+    </>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  connection: css({
+    position: 'absolute',
+    zIndex: 1000,
+    pointerEvents: 'none',
+  }),
+  vertex: css({
+    fill: '#44aaff',
```

Comment:
this probably ought to be a default theme color to "de-noise" the number of grays the user is presented with in our UI at any given time. Maybe see if any of the `theme.colors.border` options look good: https://github.com/grafana/grafana/blob/291ec08150cff7f083dfa1d997f343cee2d6f9f2/packages/grafana-data/src/themes/createColors.ts#L102-L104